[2024-05-06T13:18:12.475+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T13:08:21.959774+00:00 [queued]>
[2024-05-06T13:18:12.527+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.write_to_clean_data_task manual__2024-05-06T13:08:21.959774+00:00 [queued]>
[2024-05-06T13:18:12.528+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-05-06T13:18:12.965+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): write_to_clean_data_task> on 2024-05-06 13:08:21.959774+00:00
[2024-05-06T13:18:13.348+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main', 'write_to_clean_data_task', 'manual__2024-05-06T13:08:21.959774+00:00', '--job-id', '299', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmptjxe6byr']
[2024-05-06T13:18:13.405+0000] {standard_task_runner.py:85} INFO - Job 299: Subtask write_to_clean_data_task
[2024-05-06T13:18:13.414+0000] {standard_task_runner.py:57} INFO - Started process 171 to run task
[2024-05-06T13:18:14.239+0000] {task_command.py:415} INFO - Running <TaskInstance: main.write_to_clean_data_task manual__2024-05-06T13:08:21.959774+00:00 [running]> on host e502cb09ab6c
[2024-05-06T13:18:15.053+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='write_to_clean_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-06T13:08:21.959774+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-06T13:08:21.959774+00:00'
[2024-05-06T13:18:15.777+0000] {warnings.py:109} WARNING - /opt/***/dags/main.py:72: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql_query(query, conn)

[2024-05-06T13:19:55.286+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-05-06T13:19:55.500+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main, task_id=write_to_clean_data_task, execution_date=20240506T130821, start_date=20240506T131812, end_date=20240506T131955
[2024-05-06T13:19:55.787+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-05-06T13:19:55.915+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
